### PR TITLE
Chore/update to v8

### PR DESCRIPTION
**Pull request by antkiewicz**

### CODE REVIEW

**Comment by antkiewicz** [comment]
@NothingEverHappens Angular 8 now requires a node v10 `"node": ">=10.9.0 <11.0.0"`, CI config needs to be updated.

**Comment by kirjs** [comment]
Looks like CI is failing, something related to the node version?
Let me know if you need help figuring that out

**Comment by antkiewicz** [comment]
@kirjs Yep, Angular now requires node v10 `"node": ">=10.9.0 <11.0.0"`, but CI is configured for v8.

File: apps/angular-thirty-seconds-e2e/tsconfig.e2e.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "outDir": "../../../dist/out-tsc/apps/angular-thirty-seconds-e2e",
+    "outDir": "../../../dist/out-tsc",
```
[the remark this answers is not in the document]
Comment (antkiewicz) [reply apps/angular-thirty-seconds-e2e/tsconfig.e2e.json]: @kirjs not sure, change was made by an update script

[the remark this answers is not in the document]
Comment (thekiba) [reply apps/angular-thirty-seconds-e2e/tsconfig.e2e.json]: I think we should revert this change

File: apps/angular-thirty-seconds/src/app/create-snippet/create-snippet.component.ts [hunk 2]
```diff
@@ -29,8 +29,8 @@ function validatorMaxTags(maximumTags: number) {
 })
 export class CreateSnippetComponent {
 
-  @ViewChild('tagInput') tagInput: ElementRef<HTMLInputElement>;
-  @ViewChild('auto') matAutocomplete: MatAutocomplete;
+  @ViewChild('tagInput', { static: true }) tagInput: ElementRef<HTMLInputElement>;
```
Comment (thekiba) [inline apps/angular-thirty-seconds/src/app/create-snippet/create-snippet.component.ts]: Why `true`?

Comment (antkiewicz) [reply apps/angular-thirty-seconds/src/app/create-snippet/create-snippet.component.ts]: @thekiba Breaking changes in Angular v8 in preparation for v9: https://angular.io/guide/static-query-migration

Comment (antkiewicz) [reply apps/angular-thirty-seconds/src/app/create-snippet/create-snippet.component.ts]: @thekiba After re-reading the docs I realize a `{ static: false }` is the recommended flag. I'll update it.

**Comment by NothingEverHappens** [comment]
Amazing

**Pull request merged**